### PR TITLE
add alidns,unicom dns,google dns and increase maxLength of certificate to resolved #672 and #572

### LINKED ISSUE #572: change the default DNS resolver to `1.1.1.1` and `8.8.8.8`
https://github.com/iresty/apisix/blob/master/conf/config.yaml#L32
`114.114.114.114` is only friendly for China, PR is welcomed.

### LINKED ISSUE #672: feature:to increase maxLength of certificate
invalid configuration: invalid "maxLength" in docuement at pointer "#/cert"

### PR DESCRIPTION
**Pull request by fc13240**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/iresty/apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

add alidns,unicom dns,google dns to resolved #572
increase maxLength of certificate to resolved #672

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #572 
Fix #672

### CODE REVIEW

File: lua/apisix/schema_def.lua [hunk 1]
```diff
@@ -352,10 +352,10 @@ _M.ssl = {
     type = "object",
     properties = {
         cert = {
-            type = "string", minLength = 128, maxLength = 4096
+            type = "string", minLength = 128, maxLength = 64*1024
         },
         key = {
-            type = "string", minLength = 128, maxLength = 4096
+            type = "string", minLength = 128, maxLength = 64*1024
```
Comment (moonming) [inline lua/apisix/schema_def.lua]: `key` will not so long.

File: conf/config.yaml [hunk 2]
```diff
@@ -29,8 +29,13 @@ apisix:
   #   udp:                        # UDP proxy port list
   #     - 9200
   #     - 9211
-  dns_resolver:                   # use 114.114.114.114 as default DNS resolver, disable IPv6 and enable local DNS.
+  dns_resolver:                   # use 114.114.114.114 as default DNS resolver, disable IPv6 and enable local DNS, add alidns,unicom dns,google dns
```
Comment (moonming) [inline conf/config.yaml]: 114.114.114.114 is not the default DNS resolver now.

File: doc/plugins/jwt-auth-cn.md [hunk 3]
```diff
@@ -39,6 +39,11 @@ curl http://127.0.0.1:9080/apisix/admin/consumers -X PUT -d '
     }
 }'
 ```
+你可以使用浏览器打开 dashboard：`http://127.0.0.1:9080/apisix/dashboard/`，通过 web 界面来完成上面的操作，先增加一个 consumer：
```
Comment (moonming) [inline doc/plugins/jwt-auth-cn.md]: please modify the english version too.

**Comment by fc13240** [comment]
add alidns,unicom dns,google dns and increase maxLength of certificate to resolved #672 and #572

**Pull request merged**